rdm/pull.py

```diff
@@ -0,0 +1,56 @@
+'''
+Functions that parse the given project management service and produce
+`requirements.yml` and `problem_reports.yml` files.  The selected project
+management backend is specified in the project configuration file.
+'''
+import os
+from getpass import getpass
+
+from github import Github
+
+from rdm.util import load_yaml, write_yaml, remove_carriage_return
+
+
+def pull_requirements_and_reports(system_yml_path):
+    system_settings = load_yaml(system_yml_path)
+    output_dir = os.path.dirname(system_yml_path)
+    requirements_path = os.path.join(output_dir, 'requirements.yml')
+    problem_reports_path = os.path.join(output_dir, 'problem_reports.yml')
+
+    if system_settings['software_requirements_location'] == 'GitHub':
+        requirements, problem_reports = pull_from_github(system_settings['repository'])
+    else:
+        raise ValueError("Specified software requirements location not yet supported.")
+
+    write_yaml(requirements, requirements_path)
+    write_yaml(problem_reports, problem_reports_path)
+
+
+def pull_from_github(repository):
```

Comment:
Perhaps rename the argument: `repository_url` to distinguish it from `repository`?

Reply:
Gosh, don't know how I missed that one. Fixed!